org.opent2t.sample.thermostat.superpopular/org.opent2t.sample.thermostat.superpopular.json

```diff
@@ -91,6 +91,18 @@
       "$ref": "oic.baseResource.json#/definitions/oic.r.baseresource"
     },
     {
+      "$ref": "oic.r.mode.json#/definitions/oic.r.mode"
+    },
+    {
+      "$ref": "oic.r.clock.json#/definitions/oic.r.clock"
+    },
+    {
+      "$ref": "oic.r.temperature.json#/definitions/oic.r.temperature"
+    },
+    {
+      "$ref": "oic.r.humidity.json#/definitions/oic.r.humidity"
+    },
```

Comment:
I thought if they are referenced above they need to be here. is that the wrong assumption?

---
In reply to: [79676179](https://github.com/openT2T/translators/pull/102#discussion_r79676179) [](ancestors = 79676179)